src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrayOfMen = people.filter(person => {
+    return arguments.length < 2
+      ? person.sex === 'm'
+      : Math.ceil(person.died / 100) === century && person.sex === 'm';
```

Comment:
move `person.sex === 'm'` out of condition as it's common value `person.sex === 'm' && (century ? ... : .. )`